### ISSUE TITLE
Use m4a prior mp3, 

### ISSUE THREAD

**Issue by kotlinski** [kotlinski/dagens-eko-player#43]
Sometimes mp3 is not available, some times the m4a is not available.

`broadcast.broadcastfiles: [{url: *.m4a}]` vs `listenpodfiles: [{url: *.mp3}]`

**Comment by kotlinski** [comment]
https://www.izotope.com/en/learn/whats-the-difference-between-file-formats.html

**Comment by kotlinski** [comment]
Example of json response not having listenpodfile: 
```json
{
  "audiopreference": "default",
  "audiopresentation": "format",
  "audiopriority": "aac",
  "broadcast": {
    "availablestoputc": "/Date(1672388760000)/",
    "broadcastfiles": [
      {
        "duration": 60,
        "id": 8598620,
        "publishdateutc": "/Date(1669796700000)/",
        "statkey": "/app/avsnitt/nyheter (ekot)[k(83)]/ekot senaste nytt[p(5380)]/[e(2052644)]",
        "url": "https://sverigesradio.se/topsy/ljudfil/srapi/8598620-hi.m4a"
      }
    ],
    "playlist": {
      "duration": 60,
      "id": 8598620,
      "publishdateutc": "/Date(1669796700000)/",
      "statkey": "/app/avsnitt/nyheter (ekot)[k(83)]/ekot senaste nytt[p(5380)]/[e(2052644)]",
      "url": "https://api.sr.se/api/radio/radio.aspx?type=broadcast&id=8598620&codingformat=.m4a&metafile=asx&quality=hi"
    }
  },
  "broadcasttime": { "endtimeutc": "/Date(1669796760000)/", "starttimeutc": "/Date(1669796700000)/" },
  "channelid": 701,
  "description": "Senaste nytt varje timme från Ekoredaktionen.",
  "id": 2052644,
  "imageurl": "https://static-cdn.sr.se/images/5380/a7898d6c-786f-4fcb-b68e-c5f56f4b3bef.jpg?preset=api-default-square",
  "imageurltemplate": "https://static-cdn.sr.se/images/5380/a7898d6c-786f-4fcb-b68e-c5f56f4b3bef.jpg",
  "photographer": "",
  "program": { "id": 5380, "name": "Ekot senaste nytt" },
  "publishdateutc": "/Date(1669796700000)/",
  "title": "Nyheter från Ekot",
  "url": "https://sverigesradio.se/avsnitt/2052644"
}

```

**Comment by kotlinski** [comment]
Solved with #49
